docs/execution-providers/Xnnpack-ExecutionProvider.md

```diff
@@ -23,12 +23,14 @@ Accelerate ONNX models on Android devices and WebAssembly with ONNX Runtime and
 Pre-built packages of ONNX Runtime([`onnxruntime-android`](https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime-android)) with XNNPACK EP for Android are published on Maven.
 See [here](../install/index.md#install-on-android) for installation instructions.
 
-There is no pre-built package for iOS yet. We will update it once it is ready.
+Pre-built binaries(`onnxruntime-objc` and `onnxruntime-c`) of ONNX Runtime with XNNPACK EP for iOS are published to CocoaPods.
+See [here](../install/index.md#install-on-ios) for installation instructions.
+
```

Comment:
This won't be available until the 1.14 release will it?

Reply:
@wejoncy Can you clarify/correct this?

Reply:
Yeah, This won't be avaible until a new to support xnnpack ep is avaible. I am working on that PR.

Reply:
Sure. Added.